contracts/GovHub.template

```diff
@@ -323,7 +324,7 @@ contract GovHub is System, IParamSubscriber {
   /// @param member The address of the member to remove
   function removeMember(address member) external onlyInit onlyGov onlyIfMember(member) {
     require(memberSet.length > 5, "at least five members in DAO");
-    uint256 index = members[member];
+    uint256 index = members[member]; //@openissue
```

Comment:
highly misleading. consider using (anywhere this pattern is used) indexPlusOne. it's ugly but it clearly conveys the meaning of this value